libs/clients/health-insurance-v2/src/lib/clients-health-insurance-v2.module.ts

```diff
@@ -0,0 +1,46 @@
+import { DynamicModule } from '@nestjs/common'
+import { Configuration, DocumentApi } from '../../gen/fetch'
+import { HealthInsuranceV2Options } from './clients-health-insurance-v2.config'
+import { createEnhancedFetch } from '@island.is/clients/middlewares'
+
+export class HealthInsuranceV2Client {
```

Comment:
Is this API a superset of (or meant to replace) the existing health-insurance APIs?

Reply:
The plan is to replace the existing one once the sjukra accident notification application has been delivered.

Reply:
Can't wait! :D